tests/socketio_win32_ut/CMakeLists.txt

```diff
@@ -1,24 +0,0 @@
-#Copyright (c) Microsoft. All rights reserved.
```

Comment:
># [](start = 0, length = 1)

We cannot add comment in the metadata so I write it here the nitpick: Fix in the commit messagethe "uni"....